sparql/SPARQLParser.g4

```diff
@@ -0,0 +1,694 @@
+/*
+ * Copyright (c) 2023 by Bart Kiers
+ *
+ * The MIT license.
+ *
+ * Permission is hereby granted, free of charge, to any person
+ * obtaining a copy of this software and associated documentation
+ * files (the "Software"), to deal in the Software without
+ * restriction, including without limitation the rights to use,
+ * copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the
+ * Software is furnished to do so, subject to the following
+ * conditions:
+ *
+ * The above copyright notice and this permission notice shall be
+ * included in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+ * EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES
+ * OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+ * NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+ * HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
+ * WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+ * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+ * OTHER DEALINGS IN THE SOFTWARE.
+ *
+ * (REVISION Last updated: 18 October 2023)
+ */
+/// Grammar according the latest (October 2023) specs: https://www.w3.org/TR/sparql11-query/#grammar
+parser grammar SPARQLParser;
+
+options {
+  tokenVocab=SPARQLLexer;
+}
+
+///  [1]    QueryUnit : Query
```

Comment:
I'd also remove all comments to parser rules.

Reply:
I copied the grammar from my own repo, and I find such comments often very useful: when I encounter an error, I double check if it was my implementation that contains the error, or if it was the specs that was incomplete (or old). But no problem to strip them for this repo.

Reply:
Ok, we can leave them as it if you cosider them useful.